src/exchange/message.py

```diff
@@ -19,8 +19,10 @@ def validate_role_and_content(instance: "Message", *_: Any) -> None:  # noqa: AN
         if instance.tool_use:
             raise ValueError("User message does not support ToolUse")
     elif instance.role == "assistant":
-        if not (instance.text or instance.tool_use):
```

Comment:
llama3.1 doesn't pass yet, but it was already discussed that empty is permitted here, and raising an error like we did is worse than no error (got us thinking there was a format problem)

Reply:
ah ok - should get rid of this then?

Reply:
you mean put back in the old check as we don't suggest llama3? np